test/hotspot/jtreg/compiler/c2/TestAndConZeroCCP.java

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
The test should probably be moved to `test/hotspot/jtreg/compiler/ccp/`, that would be more specific.

Reply:
Done